<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoamerMap</class>
 <widget class="QDialog" name="RoamerMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Roamer Map</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="labelMap">
     <property name="minimumSize">
      <size>
       <width>504</width>
       <height>236</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>504</width>
       <height>236</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/images/roamers.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelEntei">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/images/entei.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Entei</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelRaikou">
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/images/raikou.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Raikou</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="labelLatias">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/images/latias.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="labelLatios">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/images/latios.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Latias / Latios</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
